README.md

```diff
@@ -407,7 +407,6 @@ nextcloud_tls_cert_method: "self-signed"
 
 Defines various method for retrieving a TLS certificate.
 -   **self-signed**: generate a _one year_ self-signed certificate for the trusted domain on the remote host and store it in _/etc/ssl_.
--   **signed**: copy provided signed certificate for the trusted domain to the remote host or in /etc/ssl by default.
   Uses:
 
 ```yaml
```

Comment:
@marioqxx just a note that this usage is not for **self-signed**. With self-signed you don't need `nextcloud_tls_src_cert` or `nextcloud_tls_src_cert_key` so the documentation was correct before with the order. Probably you just need to revert the order and change the parameters for signed related to your change.

Reply:
Hmm, not sure what happened. I adapted README, but then git push failed telling me that some other push happened. I followed the suggestion to do a 'pull config pull.rebase false' followed by 'git pull'. README was replaced and 'git push' then uploaded it on my git-fork. The README now looks correct. I hope I haven't messed anything up. **self-signed** in its current version does not have parameters.

Reply:
While checking README, I've noticed one more thing `nextcloud_tls_cert` and `nextcloud_tls_cert_key` are mentioned only in `installed` option, but you are using them in your `tls_signed.yml`. @marioqxx Could you verify where is the issue? Missing README lines or typo?

Reply:
I got all `nextcloud_tls_*` variables which you are using in your PR and here's the list for easier checking. Could you verify that all of them are OK?
```
nextcloud_tls_cert
nextcloud_tls_cert_key
cert_path
nextcloud_tls_chain_file
nextcloud_tls_src_chain
nextcloud_tls_src_cert
nextcloud_tls_src_cert_key
```

Reply:
Hi Wiktor, I hope my first comments explains this. I appended "_file", but this was not reflected in "tls_signed.yml". Now it is and it is in "default/main.yml" as well as the README consistently described. Therefore to you list the following needs to be changed: 'nextcloud_tls_cert`  to 'nextcloud_tls_cert_file` and `nextcloud_tls_cert_key` to `nextcloud_tls_cert_key_file`. All variables are only for "tls_signed.yml". For "tls_installed" we have: "nextcloud_tls_cert", "nextcloud_tls_cert_key" and "nextcloud_tls_cert_chain" where I only added tha last in this list, but did not touch/alter the naming.

Reply:
That was exactly what I meant, thanks :)